features/contact/requesting_contact.feature

```diff
@@ -8,7 +8,7 @@ Feature: Requesting contact
         Given the store operates on a single channel in "United States"
         And this channel has contact email set as "contact@goodshop.com"
 
-    @ui @api @email
+    @ui @email
```

Comment:
There is an issue with sensing contact request while being logged in, there is already a task for it

Reply:
Maybe in this way to not forget to revert it?
```suggestion
    @ui @email #api
```